trino/base/hive-metastore-dc.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hive-metastore
+  labels:
+    instance: dev-presto
+    role: hive-metastore
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      instance: dev-presto
+      role: hive-metastore
+  template:
+    metadata:
+      labels:
+        instance: dev-presto
+        role: hive-metastore
+    spec:
+      containers:
+        - resources:
+            requests:
+              cpu: 1
+              memory: 4Gi
+            limits:
+              cpu: 1
+              memory: 4Gi
+          terminationMessagePath: /dev/termination-log
+          name: hive-metastore
+          command:
+            - /opt/bin/start-hive-metastore.sh
+          livenessProbe:
+            exec:
+              command:
+                - bash
+                - '-c'
+                - netstat -lpn | grep 9083
+            initialDelaySeconds: 30
+            timeoutSeconds: 30
+            periodSeconds: 30
+            successThreshold: 1
+            failureThreshold: 5
+          env:
+            - name: HIVE_METASTORE_JDBC_URL
+              value: >-
+                jdbc:postgresql://trino-db.$(namespace).svc.cluster.local:5432/trino
+            - name: HIVE_METASTORE_DRIVER
+              value: org.postgresql.Driver
+            - name: HIVE_METASTORE_USER
+              value: trino
+            - name: HIVE_METASTORE_PASSWORD
+              value: trino
+            - name: GOOGLE_CLOUD_KEY_FILE_PATH
+            - name: S3_ENDPOINT
+              value: 'https://s3.upshift.redhat.com/'
+            - name: S3_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: aws-secret
+                  key: AWS_ACCESS_KEY_ID
+            - name: S3_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: aws-secret
+                  key: AWS_SECRET_ACCESS_KEY
+            - name: AZURE_WASB_STORAGE_ACCOUNT
+            - name: AZURE_WASB_ACCESS_KEY
+            - name: AZURE_ABFS_STORAGE_ACCOUNT
+            - name: AZURE_ABFS_ACCESS_KEY
+            - name: AZURE_ADL_CLIENT_ID
+            - name: AZURE_ADL_CREDENTIAL
+            - name: AZURE_ADL_REFRESH_URL
+            - name: REGION
+          ports:
+            - containerPort: 9083
+              protocol: TCP
+          imagePullPolicy: IfNotPresent
+          terminationMessagePolicy: File
+          image: 'quay.io/rmartine/hive-metastore:trino'
+      restartPolicy: Always
```

Comment:
I believe several of the next lines are all default values. If that's the case we don't need to (and should not) include them here.